dv/uvm/core_ibex/Makefile

```diff
@@ -426,7 +426,9 @@ fcov:
           --custom_target riscv_dv_extension
 
 # Merge all output coverage directories into the <out>/rtl_sim directory
-cov:
+#
+# Any coverage databases generated from the riscv_dv_fcov target will be merged as well.
+gen_cov:
```

Comment:
Wouldn't it make more sense for `gen_cov` to depend on `riscv_dv_fcov`? Otherwise you have to run `sim` in order to get everything to happen in the right order.

Reply:
hm, that's a good point, will update accordingly!